.rubocop_todo.yml

```diff
@@ -31,3 +31,8 @@ Metrics/MethodLength:
 # Offense count: 3
 Metrics/PerceivedComplexity:
   Max: 50
+
+# Disable this because Google::Protobuf::Map has a method 'has_key?', and we do
+# not wanna get errors like '"has_key?" is deprecated, use 'key?' instead'.
```

Comment:
This Rubocop exception has been removed after the refactor to use open class to avoid modifying the method name from key? to has_key?